src/test/java/appModules/FooterMenu_Action.java

```diff
@@ -0,0 +1,53 @@
+package appModules;
+
+import java.net.HttpURLConnection;
+import java.net.URL;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import pageObjects.Footer_Menu;
+
+public class FooterMenu_Action {
```

Comment:
Can we change the class name to VerifyLinks_Action as we can use these methods for any page ?